src/parser/jobs/blm/Leylines.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import {Rule, Requirement} from 'parser/core/modules/Checklist'
+import Module from 'parser/core/Module'
+import {ActionLink} from 'components/ui/DbLink'
+import {i18nMark, Trans} from '@lingui/react'
+
+const LEYLINE_DURATION = 30000
+
+export default class Leylines extends Module {
+	static handle = 'leylines'
+	static i18n_id = i18nMark('blm.leylines.title')
+	static title = 'Ley Lines'
+
+	static dependencies = [
+		'checklist',
+	]
+	_CircleOfPowers = {
```

Comment:
Non-blocker, but I'd recommend standardizing your variable style to keep things more clean and consistent. Half of them seem to start with upper case letters (e.g. this one) and the other half with lower (e.g. `const circleOfPowerDurations` down in `_onComplete()`). Lower is my personal preference, but all I really care about is that you adhere to a standard, whichever it may be. I put the comment here since it's the first non-static variable in the file, but it applies to all of them going down from here (including some function params).